lib/ansible/cli/galaxy.py

```diff
@@ -1638,34 +1643,42 @@ def execute_list_collection(self, artifacts_manager=None):
             validate_collection_name(collection_name)
             namespace_filter, collection_filter = collection_name.split('.')
 
-        collections = list(find_existing_collections(
+        collections = []
+        fqcn_precedence = {}
+        for collection in find_existing_collections(
             list(collections_search_paths),
             artifacts_manager,
             namespace_filter=namespace_filter,
             collection_filter=collection_filter,
             dedupe=False
-        ))
+        ):
+            collections.append(collection)
+            fqcn_precedence.setdefault(collection.fqcn, []).append(collection.src)
 
         seen = set()
         fqcn_width, version_width = _get_collection_widths(collections)
+        preferred_header = 'Preferred' if any(len(matches) > 1 for matches in fqcn_precedence.values()) else ''
```

Comment:
```suggestion
        preferred_header = 'Active' if any(len(matches) > 1 for matches in fqcn_precedence.values()) else ''
```

Reply:
Some more bikeshedding: 'Priority', 'Tier', 'Stage'...

Also wondering if a tree representation would be helpful.

Reply:
dominant, preceding, effective, active

Reply:
or we can do the opposite, mark 'masked'